### PR TITLE
[DOC] Deep revision of the documentation

### LINKED ISSUE #455: Readthedocs does not build part of the documentation
Apparently 0.9.1 did build correctly

### PR DESCRIPTION
**Pull request by oesteban**
- (Hopefully) fixes #455: write graphs of workflows
- Fixes some equations that were not rendering well for the anatomical IQMs
- Revised description of functional IQMs
- Included demo videos
- Included beautified workflows

### CODE REVIEW

File: mriqc/qc/functional.py [hunk 1]
```diff
@@ -21,32 +21,73 @@
...
-  RMS variance over voxels (``dvars``), calculated
+  RMS variance over voxels ([Power2012]_ ``dvars``) indexes the rate of change of
+  BOLD signal across the entire brain at each frame of data. DVARS is calculated
   `with nipype <http://nipype.readthedocs.io/en/latest/interfaces/generated/\
-nipype.algorithms.confounds.html#computedvars>`_ before motion correction.
-- **Ghost to Signal Ratio** (:py:func:`~mriqc.qc.functional.gsr`, ``ghost_*``:
-  along the two possible phase-encoding axes **x**, **y**.
-- **Global Correlation** (:py:func:`~mriqc.qc.functional.gcor`, ``gcor``).
-- **Temporal SNR** (:abbr:`tSNR (temporal SNR)`, ``tsnr``) is the median value
+nipype.algorithms.confounds.html#computedvars>`_ before motion correction:
```
Comment (chrisfilo) [inline mriqc/qc/functional.py]: This is after motion correction. Also mention that intensities are scaled to 1000 leading to the units being expressed in x10 % BOLD change.

File: mriqc/qc/functional.py [hunk 2]
```diff
@@ -21,32 +21,73 @@
...
+
+- **Temporal SNR** (:abbr:`tSNR (temporal SNR)`, ``tsnr``) is a simplified
+  interpretation of the tSNR definition [Kruger2001]_. We report the median value
   of the `tSNR map <http://nipype.readthedocs.io/en/latest/interfaces/generated/\
-nipype.algorithms.confounds.html#tsnr>`_.
+nipype.algorithms.confounds.html#tsnr>`_ calculated like:
+
+  .. math ::
+
+      \\text{tSNR} = \\frac{\\langle S \\rangle_t}{\\sigma_t}
```
Comment (chrisfilo) [inline mriqc/qc/functional.py]: Define the variables in the equation.

File: mriqc/qc/functional.py (hunk 1 above)
Comment (chrisfilo) [inline mriqc/qc/functional.py]: It would be also good to explain or add reference to the standardized versions of DVARS.

**Comment by oesteban** [comment]
@chrisfilo the refactoring is related to fixing the generation of the workflow graphs in the documentation. In general, it is just passing valid settings for the workflow to build correctly.

There are two minor changes to the FSL FAST interface and the SpatialNormalization regarding the selection of the modality. I removed those if ... else since just substringing would do. I hope that improves code readability.

**Comment by oesteban** [comment]
thumbs up?

**Comment by oesteban** [comment]
@chrisfilo new docs are up :) http://mriqc.readthedocs.io/en/latest/index.html

Including the workflows missing in #455.

**Pull request merged**